packages/lib-classifier/src/store/SubjectStore/SubjectStore.js

```diff
@@ -6,6 +6,7 @@ import { getIndexedSubjects, subjectSelectionStrategy } from './helpers'
 import { filterByLabel, filters } from '../../components/Classifier/components/MetaTools/components/Metadata/components/MetadataModal'
 import ResourceStore from '@store/ResourceStore'
 import Subject from './Subject'
+import ImageAndTextSubject from './ImageAndTextSubject'
 import SingleImageSubject from './SingleImageSubject'
 import SingleTextSubject from './SingleTextSubject'
 import SingleVideoSubject from './SingleVideoSubject'
```

Comment:
Not required for this PR, but with the number of subject models growing, we could move them into their own folder eg.
```js
import * as SubjectModels from './subjects'
```
or similar.

That would save some boilerplate code whenever a new subject type is added.

Reply:
I think that's a good idea, opened #3041 to track, it would definitely clean things up in a nice way and help going forward as more subjects are added